### ISSUE TITLE
Please sign assembly with a strong name

### ISSUE THREAD

**Issue by junrau** [gmamaladze/globalmousekeyhook#49]
The application I use this NuGet package in has strongly named assemblies, so when I update this package, I have to manually sign the assembly.  Not difficult, but it would be nice if it came signed.

**Comment by cscorley** [comment]
Also facing this issue trying to update to v5.4.0 from v5.3.0

**Comment by claudiospizzi** [comment]
Yeah, that would be really great, if you can sign the assembly in the next release!

**Comment by sguergachi** [comment]
Facing this issue as well

**Comment by Khaos66** [comment]
Please publish a signed version

**Comment by Khaos66** [comment]
nvm... just found this blog fixing the strong name issue easily: https://www.nuget.org/packages/strongnamer
